src/material.js

```diff
@@ -80,7 +80,8 @@ export class Material {
         return {
             lattice: this.Lattice.toJSON(),
             basis: this.Basis.toJSON(),
-            name: this.name || this.formula
+            name: this.name || this.formula,
+            isNonPeriodic: this.isNonPeriodic || false
```

Comment:
This function fixes the cloning issue inside of https://github.com/Exabyte-io/materials-designer/pull/29/files#diff-07117e6feb13357543259d645dcae43645f34789d559cfebdcd62a99e17c3604